aws/rust-runtime/aws-inlineable/Cargo.toml

```diff
@@ -12,15 +12,16 @@ publish = false
 repository = "https://github.com/awslabs/smithy-rs"
 
 [dependencies]
+aws-credential-types = { path = "../aws-credential-types" }
 aws-endpoint = { path = "../aws-endpoint" }
 aws-http = { path = "../aws-http" }
-aws-sig-auth = { path = "../../rust-runtime/aws-sig-auth" }
+aws-sig-auth = { path = "../aws-sig-auth" }
```

Comment:
How did this work before?

Reply:
Before and after both point to the same path. It's just that the way it reached `aws-sig-auth` before was a bit roundabout.
```
aws/rust-runtime/aws-inlineable/../../rust-runtime/aws-sig-auth
```
vs.
```
aws/rust-runtime/aws-inlineable/../aws-sig-auth
```